upi/metal/main.tf

```diff
@@ -185,7 +185,7 @@ resource "aws_route53_record" "ctrlp_int" {
 }
 
 resource "aws_route53_record" "apps_noworker" {
-  count = var.worker_count == 0 ? 1 : 0
+  count = var.worker_count < 1 ? 1 : 0
```

Comment:
does `== 0` not work anymore?

Reply:
After we moved to terraform version 0.12, the earlier fix stopped working.

Reply:
`== "0"` would probably have worked but this solution seemed more in line with apps_worker resource below